### PR TITLE
Static pages update to use design system classes

### PR DESCRIPTION
**Pull request by timwright12**
## Description
Replacing forms library Sass with design system classes as part of the static pages isolation work. Leaving the `sip-application-status` in the HTML because there's a unit test wired up to it.

## Original issue(s)
department-of-veterans-affairs/va.gov-team#32796

### CODE REVIEW

**Review by tmitchellgcio — APPROVED** [review]

File: src/platform/forms/save-in-progress/ApplicationStatus.jsx [hunk 1]
```diff
@@ -137,7 +137,7 @@ export class ApplicationStatus extends React.Component {
           .format('MMMM D, YYYY [at] h:mm a');
 
         return (
-          <div className="usa-alert usa-alert-info background-color-only sip-application-status">
+          <div className="usa-alert usa-alert-info background-color-only sip-application-status vads-u-margin-bottom--2 vads-u-margin-top--0">
```
Comment (jerekshoe) [inline src/platform/forms/save-in-progress/ApplicationStatus.jsx:140]: Does the `sip-application-status` class still need to be here? It looks like you deleted it above

**Review by alexandec — APPROVED** [review]

**Review by jerekshoe — APPROVED** [review]

File: src/platform/forms/save-in-progress/ApplicationStatus.jsx [hunk 2]
```diff
@@ -100,7 +100,7 @@ export class ApplicationStatus extends React.Component {
         : `Deleting your ${appType}.`;
 
       return (
-        <div className="sip-application-status">
+        <div className="sip-application-status vads-u-margin-bottom--2 vads-u-margin-top--0">
```
Comment (bkjohnson) [inline src/platform/forms/save-in-progress/ApplicationStatus.jsx:103]: Should the margin-bottom be 1 instead of 2?

Comment (timwright12) [reply src/platform/forms/save-in-progress/ApplicationStatus.jsx:103]: I got .vads-u-margin-bottom--2: 16px / 1em from https://design.va.gov/utilities/margins

Comment (jerekshoe) [reply src/platform/forms/save-in-progress/ApplicationStatus.jsx:103]: I think 1em is 16px

Comment (bkjohnson) [reply src/platform/forms/save-in-progress/ApplicationStatus.jsx:103]: That's a bit confusing, but it looks like the end result is right:

image

**Review by bkjohnson — APPROVED** [review]

**Pull request merged**